django_setup_configuration/contrib/sites/steps.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.sites.models import Site
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError, transaction
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.contrib.sites.models import SitesConfigurationModel
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+
+class SitesConfigurationStep(BaseConfigurationStep):
+    config_model = SitesConfigurationModel
+    verbose_name = "Sites configuration"
+
+    namespace = "sites_config"
+    enable_setting = "sites_config_enable"
+
+    @transaction.atomic()
+    def execute(self, model: SitesConfigurationModel) -> None:
+        if not model.items:
+            raise ConfigurationRunFailed("Please specify one or more sites")
+
+        # We need to ensure the current site is updated, to make sure that `get_current`
+        # keeps working
+        first_site = model.items[0]
```

Comment:
Posting the `get_current` [function of django here for reference](https://github.com/django/django/blob/main/django/contrib/sites/models.py#L48). I think we should reconsider picking the first item out of the list here and letting the user specify it in the yaml with a separate value, so the yaml might look like this:

```yaml
sites_config_enable: true
sites_config:
  current_site: domain.local2:8000
  items:
  - domain: domain.local1:8000
    name: Domain1
  - domain: domain.local2:8000
    name: Domain2
```

Afterwards you could call the `get_current` function and update the site with the site specified in the yaml.